src/mscorlib/src/System/IO/IOException.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*============================================================
+**
+** 
+** 
+**
+**
+** Purpose: Exception for a generic IO error.
+**
+**
+===========================================================*/
+
+using System;
+using System.Runtime.Serialization;
+
+namespace System.IO
+{
+    public class IOException : SystemException
```

Comment:
This file has moved to shared partition. You should do the fix there, and not adding a dead copy of the file here.